common/pulp/common/constants.py

```diff
@@ -59,13 +59,13 @@
 # Constant used to determine whether a CeleryBeatLock should be removed due to age
 CELERYBEAT_LOCK_MAX_AGE = 200
 # The amount of time in seconds before a Celery process is considered missing
-CELERY_TIMEOUT_SECONDS = 300
+CELERY_TIMEOUT_SECONDS = 60
```

Comment:
Currently the wait time for pulp-manage-db is calculated by CELERY_TIMEOUT_SECONDS - currtime - last_hearbeat if we leave this to be 300 then the user has a potentially very long time to wait. I'll file a new feature with the timeout improvement and update this PR with a separate commit since it's a big change and should get it's own line in the release note.